src/ROUTES.js

```diff
@@ -6,6 +6,8 @@ export default {
     SETTINGS: '/settings',
     NEW_GROUP: '/new/group',
     NEW_CHAT: '/new/chat',
+    IOU_REQUEST_MONEY: '/iou/request',
+    IOU_BILL_SPLIT: '/iou/split',
```

Comment:
NAB, might just make these `IOU_REQUEST` and `IOU_SPLIT` for consistency with the `NEW_*` constants.